salsah/src/public/js/jquery.searchbox.js

```diff
@@ -185,6 +185,8 @@
 //						position: 'absolute',
 						left: pos.left,
 						top: pos.top +$this.outerHeight(),
+						background: 'white',
+						'z-index': 2, /* :( i don't like black magic numbers or const */
```

Comment:
The underlying buttons remains on top of the menu (not only visible, also selectable, which is a problem to select the "drop down" search list items), see animated gif on #546 .  

So we have to set a `z-index`to bring the menu items on top.  

And here I set `2` which is a _black magic number_ (like "white" [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) but more dangerous and error prone because of the esoteric, arbitrary and non documented picking).

At other places in the code, other black magic values where chosen (55 in `jquery.editvalue.js` or `jquery.tableedit.js`, 1000 in `jquery.valcomment.js`or `jquery.geonames.js`).